src/Scoreboard.cpp

```diff
@@ -288,15 +444,17 @@ cScoreboard::cScoreboard(cWorld * a_World) : m_World(a_World)
 
 cObjective * cScoreboard::RegisterObjective(const AString & a_Name, const AString & a_DisplayName, cObjective::eType a_Type)
 {
+	if (m_Objectives.count(a_Name) > 0)
+	{
+		return nullptr;
```

Comment:
Shouldn't it return the existing objective instead?

Reply:
Maybe, I don't think so though, for two reasons:
1) If someone creates a new objective, they probably expect it to not have any scores set (so if they create it and add 1 to someone's score, they would be surprised), and
2) If the objective is already created, it's likely that it's being used somewhere else to track something else, so there could be a conflict between the two uses of the scoreboard (imagine if two plugins both used a scoreboard with the same name, for whatever reason).